source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,135 @@
+.. _php-cluster-monitoring:
+
+==================
+Cluster Monitoring
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, server, topology
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to use the {+php-library+} to monitor topology
+events in a MongoDB instance, replica set, or sharded cluster. The
+driver creates topology events, also known as Server Discovery and
+Monitoring (**SDAM**) events, when there are any changes in the state of the
+MongoDB instance or cluster that you are connected to.
+
+You might use information about topology events in your
+application to understand cluster changes, assess cluster health, or
+perform capacity planning.
+
+Subscribe to Events
+-------------------
+
+You can access details about SDAM events by subscribing to them
+in your application. To subscribe to any event, create a class that
+implements the ``MongoDB\Driver\Monitoring\SDAMSubscriber`` interface,
+then use the ``MongoDB\Client::addSubscriber()`` method to register the
+event subscriber with your ``MongoDB\Client`` instance.
+
+The following code creates the ``MySubscriber`` class that implements
+``SDAMSubscriber`` to output a message when a ``ServerOpeningEvent`` is
```

Comment:
I: https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/

```suggestion
The following code creates the ``MySubscriber`` class, which implements
``SDAMSubscriber`` to output a message when a ``ServerOpeningEvent`` is
```

Reply:
reworded